packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json

```diff
@@ -120,6 +120,8 @@
   "ovhManagerSupport_new_creation_success": "Votre ticket numéro <a ui-sref=\"{{ state }}\">{{ id }}</a> a bien été pris en compte.",
   "ovhManagerSupport_new_creation_success_back": "Vous pouvez retrouver le fil de discussion de ce ticket depuis la <a ui-sref=\"{{ state }}\" ui-sref-opts=\"{ reload: true }\">liste des tickets</a>.",
   "ovhManagerSupport_new_creation_error": "Une erreur est survenue lors de la création de votre ticket.",
+  "ovhManagerSupport_new_creation_error_already_opened": "Un ticket d'intervention est déjà en cours pour ce serveur.",
```

Comment:
how do you know that the ticket is about a dedicated server ?

Reply:
Is about support-app

Reply:
Right